.travis.yml

```diff
@@ -1,9 +1,31 @@
-language: ruby
-cache: bundler
-rvm:
-  - 2.2.5
-sudo: false
-branches:
-  only:
-    - master
-bundler_args: --without=integration
+# Use Travis's cointainer based infrastructure
+sudo: required
+services: docker
+
+addons:
+ apt:
+   sources:
+     - chef-stable-precise
+   packages:
+     - chefdk
```

Comment:
We're installing the `chefdk` to have access to `cookstyle`, `foodcritic`, `rspec` and `kitchen` in the build.

Kitchen itself will install ChefDK in the Docker container where it runs the tests.